src/main/cpp/systemds.cpp

```diff
@@ -192,18 +180,18 @@ JNIEXPORT jint JNICALL Java_org_apache_sysds_utils_NativeHelper_dconv2dBiasAddDe
   if(inputPtr == NULL || biasPtr == NULL || filterPtr == NULL || retPtr == NULL)
     return (jint) -1;
   
-  int nnz = dconv2dBiasAddDense(inputPtr, biasPtr, filterPtr, retPtr,
-    (int) N, (int) C, (int) H, (int) W, (int) K, (int) R, (int) S,
-    (int) stride_h, (int) stride_w, (int) pad_h, (int) pad_w, (int) P, (int) Q, true, (int) numThreads);
+  size_t nnz = dconv2dBiasAddDense(inputPtr, biasPtr, filterPtr, retPtr, (int) N, (int) C, (int) H, (int) W, (int) K,
+								   (int) R, (int) S, (int) stride_h, (int) stride_w, (int) pad_h, (int) pad_w, (int) P,
+								   (int) Q, true, (int) numThreads);
```

Comment:
indent

Reply:
Mental note: make a clang-tidy config to aid proper formatting. A plethora of editors and IDEs are in use (Visual Studio, Eclipse, IntelliJ, CLion, QT Creator, Kate, Vim, Gedit, Notepad++ all had a go at these files 😨 )